sql/log_event.h

```diff
@@ -3684,6 +3690,13 @@ class Gtid_log_event: public Log_event
 
   bool is_valid() const override
   {
+#ifdef WITH_WSREP
+    if (skip_gtid_seqno_check)
```

Comment:
@hemantdangi-gc Instead of adding a new variable, can't we directly do wsrep_skip_gtid_seqno logic here?